src/partition/repart.c

```diff
@@ -3512,21 +3512,6 @@ static int context_write_partition_table(
 
         assert(context);
 
-        if (arg_pretty > 0 ||
-            (arg_pretty < 0 && isatty(STDOUT_FILENO) > 0) ||
-            !FLAGS_SET(arg_json_format_flags, JSON_FORMAT_OFF)) {
-
-                (void) context_dump_partitions(context, node);
-
-                if (arg_json_format_flags & JSON_FORMAT_OFF) {
-                        putc('\n', stdout);
-                        (void) context_dump_partition_bar(context, node);
-                        putc('\n', stdout);
-                }
-
-                fflush(stdout);
-        }
```

Comment:
this doesn't look right. the idea was to show this before we do anything, and in particular show this if we are in dry run mode, so that you can verify everything is in order before you try again. Given that the dry-run check is now done *before* we dump the table the dry-run mode is half useless...

Reply:
You are right. But also, it's a bit strange if we first say "Applying changes" and only much later say which changes... Sounds better to say first what we are going to change, before we change it.

Reply:
(I mean, we had various bugs where the application of the changes actually failed, so I think it's kinda nice to report first what are going to do, so that this is visible once we fail)